adagrams/game.py

```diff
@@ -88,35 +93,37 @@ def get_highest_word_score(word_list: list) -> tuple[str, int]:
     #use tiebreaker func if there are ties
     if len(ties) > 0:
         winning_word = tiebreaker(ties)
-        winning_score = score_tracker[winning_word]
+        # winning_score = score_tracker[winning_word]
     
     return (winning_word, winning_score)
 
 
-def tiebreaker(tied_list: list[str]) -> str:
-    shortest_len_word = None
+def tiebreaker(tied_list):
+    # shortest_len_word = None
+    shortest_word = tied_list[0]
 
     for tied_words in tied_list:
-        shortest_len_word = tied_list[0]
+        # shortest_word = None
+        
         #even if there are multiples w length of 10, the first in the list will be the winner
         if len(tied_words) == 10:
             winner = tied_words
             break
         else:
-            if len(tied_words) < len(shortest_len_word):
-                shortest_len_word = tied_words
-                winner = shortest_len_word
-                break
-            if len(tied_words) == len(shortest_len_word): 
-                if tied_list.index(tied_words) < tied_list.index(shortest_len_word):
+            if len(tied_words) < len(shortest_word):
+                shortest_word = tied_words
+                winner = shortest_word
+                # break
```

Comment:
Prefer to remove logic over commenting it out. Commented logic left in code has a tendency to become uncommented as later devs come along and may get confused about whether the code was intentionally commented.